noxfile.py

```diff
@@ -172,9 +175,10 @@ def flake8(session: nox.sessions.Session):
     # Pip install the session requirements.
     session.install("flake8")
     # Execute the flake8 linter on the package.
-    session.run("flake8", PACKAGE)
-    # Execute the flake8 linter on this file.
-    session.run("flake8", __file__)
+    session.run("flake8", str(PACKAGE))
```

Comment:
`PACKAGE` is already a `str`. What's up with wrapping it again?

Reply:
Yup, I refactored and pushed that up